repos/system_upgrade/common/models/defaultgrubinfo.py

```diff
@@ -0,0 +1,15 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemFactsTopic
+
+
+class DefaultGrub(Model):
+    topic = SystemFactsTopic
+
+    name = fields.String()
+    value = fields.String()
+
+
+class DefaultGrubInfo(Model):
+    topic = SystemFactsTopic
+
+    default_grub_info = fields.List(fields.Model(DefaultGrub))
```

Comment:
we should keep every new model pretty wel covered by docstring (includes docstring for the fields) + I think that these models could be designed differently. Let's discuss it as it's highly possible I missed something, but if we can do it better, I think we should.

Reply:
we will just add docstrings here